templates/todo/projects/csharp-cosmos-sql-func/.vscode/tasks.json

```diff
@@ -0,0 +1,101 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Start API",
+            "type": "dotenv",
+            "targetTasks": "API dotnet run",
+            "file": "${input:dotEnvFilePath}"
+        },
+        {
+            "label": "API dotnet run",
+            "detail": "Helper task--use 'Start API' task to ensure environment is set up correctly",
+            "type": "func",
+            "command": "host start --port 3100 --cors *",
```

Comment:
Will this automatically `dotnet restore` and `dotnet build` the API project?

If not, you might want to add "Build API" tasks to "Start API" `targetTasks` list.